File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/AnoncredsNotIssuedException.java
package org.hyperledger.indy.sdk;

public class AnoncredsNotIssuedException extends IndyException
{
	private final static String message = "The anoncreds is not issued.";

	AnoncredsNotIssuedException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/AnoncredsNotIssuedException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when an anoncreds is not issued.
    /// </summary>
    public class AnoncredsNotIssuedException : IndyException
    {
        const string message = "The anoncreds is not issued.";

        /// <summary>
        /// Initializes a new AnoncredsNotIssuedException.
        /// </summary>
        internal AnoncredsNotIssuedException() : base(message, (int)ErrorCode.AnoncredsNotIssuedError)
        {

        }
    }

}
